shared/src/main/scala/amf/client/remod/amfcore/config/AMFLogger.scala

```diff
@@ -1,6 +1,6 @@
 package amf.client.remod.amfcore.config
 
-private[remod] class AMFLogger {
+private[amf] class AMFLogger {
```

Comment:
I would probably add a TODO in all the classes that will be public in the future so that we can clearly see it when reviewing.

Reply:
I'm adding "//TODO: ARM" as prefix on comments to know that something there has to be deleted.